### DISCUSSION TITLE
I didn't receive the license. Can anyone help me?

### DISCUSSION THREAD

**Discussion by tiaagomiguel** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#591] [Q&A]
Hello everyone!

I would like to contact someone responsible for the project, as I did not receive my license key after purchasing

thanks

**Comment by tiaagomiguel** [comment]
@davidwernhart @MatthiasKerbl can you help me?

**Reply by MatthiasKerbl** [reply]
Hi @tiaagomiguel,

Please send me the email address you used to purchase AlDente Pro to
I will look into what went wrong with your purchase.

Cheers,
Matthias

**Reply by tiaagomiguel** [reply]
Hello @MatthiasKerbl, already sent. 

if you can help me i would be very grateful,

Thank you

**Reply by MatthiasKerbl** [reply]
Unfortunately, you misspelled your email address. However, I have forwarded the purchase emails to your correct email address.

Cheers,
Matthias
